backend/src/backend/routes/users.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+from uuid import UUID
+
+from fastapi import APIRouter, Depends
+from httpx import codes
+from pydantic import BaseModel
+from sqlalchemy.orm import Session
+
+from backend.databse import get_session
+from backend.databse.models import User
+
+router = APIRouter()
```

Comment:
You should define a prefix on the router so that all routes are defined relative to that prefix